index.js

```diff
@@ -1 +1,101 @@
-console.log('hello world');
+import {
+  add, complete, remove, read, editOneElement,
+} from './jsonFileStorage.js';
+import getDateNow from './date.js';
+
+if (process.argv[2] === 'show') {
```

Comment:
you could create a variable to name `process.argv[2]` and `process.argv[3]` here to

1.  Give more context on what that is
2. Reduce repetition by using the variable instead